README.md

```diff
@@ -49,7 +49,7 @@ There are a few labels in particular that are very useful if you are new to Zen:
 * There is a [critical bugs milestone](https://github.com/CoderDojo/community-platform/milestones/Critical%20bugs) for the most important bugs.
 * Some bugs are smaller CSS issues that are marked low priority and so are not as important to work on. 
 
-### Priorities for the project
+### Priorities for the projectl
```

Comment:
`git commit --amend --noedit` is your friend (After you `git add` this file again following the change you make)